SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -203,12 +214,6 @@ Interest InterestFactory::CreatePartitionInterest(const UAbstractLBStrategy* LBS
 void InterestFactory::AddLoadBalancingInterestQuery(const UAbstractLBStrategy* LBStrategy, VirtualWorkerId VirtualWorker,
 													Interest& OutInterest) const
 {
```

Comment:
Where/how is this done now?

Reply:
Oh `SpatialVirtualWorkerTranslationManager::SpawnPartitionEntity`. I preferred how all interest management was previously done in a single file.

Reply:
Is there a reason to keep this function stub around?

Reply:
The stub is not needed.
Interest management in a single file is a bit tedious, having to keep backpointers everywhere, or bringing the interest factory along.
I had to slice it to be able to use it outside of an UnrealWorker, just because I needed the ServerNonAuthInterestResultType, but did not have a PackageMapClient.
Which, if I look at it, does not look that complicated anymore... so we could even slice the class even further.
If I'm not wrong, the main reason we did this was because CreateAndCacheInterestState actually computed huge component lists, which is not the case anymore.